dotCMS/src/integration-test/java/com/dotmarketing/portlets/browser/BrowserUtilTest.java

```diff
@@ -0,0 +1,456 @@
+package com.dotmarketing.portlets.browser;
```

Comment:
Need to add this test to the MainSuite.class

Reply:
done https://github.com/dotCMS/core/pull/21224/commits/9785d32e2b7e3e0ba28f25ba0594735ebfb4379a